common/api/core-common.api.md

```diff
@@ -3169,6 +3173,8 @@ export interface FeatureAppearanceProps {
     emphasized?: true;
     ignoresMaterial?: true;
     linePixels?: LinePixels;
+    lineRgb?: RgbColorProps | false;

```

Comment:
do we really need to type this as false?
wouldnt it being undefined be enough to do whatever you have for the "false" behavior?

Reply:
Did you read the documentation comments?

Reply:
sorry i did not earlier. i now see this is the best way to not break users and be more flexible